src/api/apollo.js

```diff
@@ -19,30 +14,26 @@ export default function createApolloClient({
 	appConfig,
 	cookieStore,
 	kvAuth0,
-	types = [],
+	types,
 	uri,
 	fetch
 }) {
-	const cache = new InMemoryCache({
-		fragmentMatcher: new IntrospectionFragmentMatcher({
-			introspectionQueryResultData: {
-				__schema: { types }
-			}
-		}),
-		// Return a custom cache id for types that don't have an id field
-		dataIdFromObject: object => {
```

Comment:
@edwinf-barreram @dyersituations I suspect we'll need something like lines 26-41 that were removed here particularly this method which ensures there is an id on every object to associate in the cache. The simplified caching mechanism for the server may also be important – if it's still available.

Reply:
Yeah, will take a full review about this!

Reply:
The simplified server caching mechanism is still available (see work in CPS), and this data ID method won't necessarily work 100% in the new Apollo Client. I had to tweak it a bit for the `lend` object, so I wonder if the new version has adjusted the caching mechanism a bit. But this would be part of the solution. @edwinf-barreram Take a look at my recent PRs to CPS for some Apollo-related work. You won't be able to copy and paste, but it might give some ideas.

Reply:
@dyersituations Thank you very much!